dom.bs

```diff
@@ -1138,9 +1196,15 @@ invoked, must run these steps:
    <var>listener</var>'s <b>capture</b> is true, terminate these substeps (and run them for the next
    <a>event listener</a>).
 
+   <li>If <var>listener</var>'s <b>passive</b> is true, set <var>event</var>'s <a>passive flag</a>.
```

Comment:
I thought this is what we agreed on [here](https://github.com/RByers/EventListenerOptions/issues/2#issuecomment-120755014) and summarized [here](https://github.com/RByers/EventListenerOptions/issues/2#issuecomment-125986241)?

I.e. calls to `preventDefault` from a passive listener never has any impact (since `passive` is a promise that the listener doesn't need to cancel).  But `Event.cancelable` is immutable and so can be changed only by code invoking dispatch based on the observation of ALL listeners.